jobs/JREGIONAL_PROCESS_RADARREF

```diff
@@ -78,7 +78,7 @@ mkdir_vrfy -p ${workdir}
 #-----------------------------------------------------------------------
 #
 $SCRIPTSDIR/exregional_process_radarref.sh \
-  CYCLE_DIR="${CYCLE_DIR}" cycle_type="${CYCLE_TYPE}" WORKDIR="${workdir}" || \
+  CYCLE_DIR="${CYCLE_DIR}" cycle_type="${CYCLE_TYPE}" WORKDIR="${workdir}" THINNING="${THINNING}" || \
```

Comment:
I would prefer RADAR_REF_THINNING instead of "THINNING"

Reply:
Changed all  THINNING  to RADAR_REF_THINNING

Reply:
Thanks! Since RADAR_REF_THINNING is now defined in var_defn.sh and can be directly accessed by JJOB and scripts, do you still want to pass this variable through an unnecessary complicated chain (i.e. ush/generate_FV3LAM_wflow.sh -> xml -> JJOB-> script)?

Reply:
Yes, this chain meets NCO standard.